src/plone/app/mosaic/browser/static/mosaic-grid.css

```diff
@@ -187,10 +160,10 @@ body.mosaic-enabled.no-portlet-columns .outer-wrapper > .container .mosaic-grid-
   padding-left: 15px;
   padding-right: 15px;
 }
-@media (min-width: 992px) {
+@media (min-width: 768px) {
```

Comment:
shouldn't we better use the variables from Barceloneta here, like defined in `variables.plone.less`?
```
@plone-screen-xs-min:              480px;
@plone-screen-sm-min:              768px;
@plone-screen-md-min:              992px;
@plone-screen-lg-min:              1200px;
```

Reply:
good question ... I didn't realize, that its not ... but extending from bootstrap css classes is a *pita* because all the media selectors are merged too ... I think I revert this whole thing and do some more improvements.

Reply:
ah sorry ... but that here is another thing. this is a small fix for `width-three-quarter` to wrap correctly in screen-sm size because `width-quarter` already did. see https://github.com/plone/plone.app.mosaic/blob/master/src/plone/app/mosaic/browser/static/css/mosaic.grid.less#L110

Reply:
ok, probably a new issue/PR worth.